### DISCUSSION TITLE
When does the app ask for width of bike and foot paths?

### DISCUSSION THREAD

**Discussion by dealerofsalvation** [streetcomplete/StreetComplete#5249] [Feedback]
Hi, in my settings I have activated the question "How wide is the bicycle path here?".
However I haven't yet been asked this question on a bike path. I've verified on Vespucci that this information hasn't been entered yet.
What could be the reason I'm not asked? Does it matter if the path is a combined cycling/foot path?

**Comment by Helium314** [comment]
see https://github.com/streetcomplete/StreetComplete/blob/e300eaab151c413c387e9a9e936e2b9013ad5dab/app/src/main/java/de/westnordost/streetcomplete/quests/width/AddCyclewayWidth.kt#L17C1-L35C53
essentially the cycleway must not allow pedestrians, or be segregated

**Reply by dealerofsalvation** [reply]
Thanks. Is it promising to request to have that extended to non-segregated paths? I mean it makes sense to record this data so that it's possible to query paths that do not meet the minimum standards. Yes I have a laser rangefinder.

**Reply by mnalis** [reply]
There were (probably :smiley:)  reasons why it was done this way. 

Whether it was just to reduce sheer number of quests only to those most needed  (e.g. if you ask width on all non-segregated cycleways, you should likely ask it on all regular footways too) or if there are more reasons (confusion what exactly is being measured?) I'm not sure.

If you are using [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) ("expert edition") fork of StreetComplete you can always remove that restriction on segregation for yourself. 
Whether such change would be suitable for general StreetComplete population, I'm not sure. You can try requesting it, but do provide good rationale why you think it would be good idea (e.g. effort / utility ratio) to ask all users more quests.

**Reply by dealerofsalvation** [reply]
Thanks very much for your reply and for pointing out SCEE, I'll certainly try it out.
